README.md

```diff
@@ -1,11 +1,6 @@
 <a name="readme-top"></a>
 
```

Comment:
- [ ] IMPORTANT:  I couldn't find a link for the backend repo.  I looked at the readme file, the review request, and the pull request description.

This means I cannot review the backend part of the project, or its functionality since I can't run it locally without the backend part.

Kindly, include a link for the backend part of your project on your readme. 👍